models/app.model.js

```diff
@@ -8,4 +8,20 @@ exports.selectTopics = () => {
         }
         return rows;
     });
+};
+
+//STILL NEED TO DO TASK 4!
+
+exports.selectArticles = () => {
+    return db.query(`SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url,
+        COUNT(comments.comment_id) AS comment_count
+        FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id
+        GROUP BY articles.article_id
+        ORDER BY articles.created_at DESC;`)
+        .then(( { rows }) => {
+            return rows.map((article) => ({
+                ...article,
+                comment_count: Number(article.comment_count),
```

Comment:
This could be done using SQL on line 17. This would be preferable and would do away with the map here.